llvm/test/tools/yaml2obj/XCOFF/dwarf2.yaml

```diff
@@ -0,0 +1,104 @@
+## Test that we can generate and read DWARF section headers
```

Comment:
This test should probably be folded into the other test case. Also, the comment should actually describe what's interesting about this test case, since the first case is actually checking an error and is not about whether we can "generate and read DWARF section headers."